<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEdit</class>
 <widget class="QWidget" name="DialogEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Задача</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="label_main">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Редактирование задачи</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Дата создания:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QDateEdit" name="dateEdit_create">
      <property name="displayFormat">
       <string>dd.MM.yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2016</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Текст задачи:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Критерий:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_criterion"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Срок исполнения:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QDateEdit" name="dateEdit_end">
      <property name="displayFormat">
       <string>dd.MM.yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2017</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Ответственный:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Дата завершения:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QDateEdit" name="dateEdit_finish">
      <property name="displayFormat">
       <string>dd.MM.yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>Принять</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QComboBox" name="lineEdit_man">
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Александр</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Алексей</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Василий</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Виктория</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Гавриил</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Дарья</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Иван</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Катерина</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Наталья</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Николай</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ольга</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Степан</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
